docs/source/refs/additional_resources.rst

```diff
@@ -0,0 +1,32 @@
+Additional Resources
+======================
+
+Here we provide external links to tools and various resources that you may also find useful.
+
+
+Sim-to-Real Resources
+------------------------
+One of the core goals of the broader Isaac project is to bring real robots to life through the power of NVIDIA technology. There are many ways to do this, and thus, many tools that you could use.  These resources are dedicated to helping you navigate these possibilities by providing examples and discussions about closing the Sim-to-Real gap and deploying policies to actual real robots.
+
+* `Closing the Sim-to-Real Gap: Training Spot Quadruped Locomotion with NVIDIA Isaac Lab <https://developer.nvidia.com/blog/closing-the-sim-to-real-gap-training-spot-quadruped-locomotion-with-nvidia-isaac-lab/>`_ is a detailed guide for training a quadruped locomotion policy for the Spot Quadruped from Boston Dynamics, and deploying it to the real robot.
+
+* `Fast-Track Robot Learning in Simulation Using NVIDIA Isaac Lab <https://developer.nvidia.com/blog/fast-track-robot-learning-in-simulation-using-nvidia-isaac-lab/>`_ is a blog post covering the gamut of features for accelerated robot learning through Isaac Lab.
```

Comment:
I wonder if we should switch this one with the Deploying Policies in Isaac Sim one. This blog seems to be more about features in Isaac Lab and relevant projects, while the Deploying Policies one is more relevant to sim-to-real or sim-to-sim deployment

Reply:
my understanding of the Deploying Policies document is that it's about deploying policies into isaac-sim, not out of